recipes/go/build.sh

```diff
@@ -0,0 +1,24 @@
+# First, build go1.4 using gcc, then use that go to build go>1.4
+# Should I make this a separate package called go-bootstrap?
+# Should I check the checksum of this tarball? What about windows?
+mkdir go-bootstrap && pushd $_
```

Comment:
There was some work done by @janschulz included in PR ( https://github.com/conda-forge/staged-recipes/pull/644 ) to get Windows to work. I think it was incomplete. Maybe he can provide you some more details about what still needed to be done and/or what issues were outstanding.

Reply:
I'll work on Windows later today. Do you might if I add you and @janschulz as maintainers to the recipe?